src/Http/Validation/Rules/Min.php

```diff
@@ -4,9 +4,15 @@
 
 namespace Engelsystem\Http\Validation\Rules;
 
+use Respect\Validation\Rules\AbstractEnvelope;
 use Respect\Validation\Rules\Min as RespectMin;
 
-class Min extends RespectMin
+class Min extends AbstractEnvelope
 {
     use StringInputLength;
+
+    public function __construct(mixed $minValue)
+    {
+        parent::__construct(new RespectMin($minValue));
```

Comment:
Min validation funktioniert noch nicht, man kann z.B. Passwörter mit der Länge 3 setzen.

Reply:
@msquare in welchem formular hattest du das getestet?